### PR TITLE
feat: Grants Tweaks 

### LINKED ISSUE #6883: Grants Tweaks
Screen Shot 2020-06-15 at 11 10 19 PM

When I completed funding of a grant, this screen should have a button for me to fund another grant. I'm left here after tweeting about the grant.

-----------

Needs some clean up so the message doesn't seem scary/warning like.
Screen Shot 2020-06-15 at 11 00 48 PM

Screen Shot 2020-06-16 at 12 24 31 AM

-----------

Phone number verification screen:
When I entered my phone number, it didn't recognize that it's a U.S.A number and the flag stayed on another country. Number also messed up when I changed the flag to U.S.A

-----------

Screen Shot 2020-06-15 at 11 53 07 PM
Change font color to Gitcoiin green #25E899 to highlight the amount this round.

### PR DESCRIPTION
**Pull request by developerfred**
## Fixes
Screen Shot 2020-06-16 at 20 13 13
Screen Shot 2020-06-16 at 20 12 35

##### Refers/Fixes
#6883
- [x]  Change font color to Gitcoin green #25E899 to highlight the amount this round.
- [x]   When I completed funding of a grant, this screen should have a button for me to fund another grant. I'm left here after tweeting about the grant.
- [x]   [Verifiy to Verify](https://github.com/gitcoinco/web/issues/6860#issuecomment-644245348)
- [x]   Phone Number Modal
- [x]   Phone number verification screen:

##### Testing

https://share.getcloudapp.com/4guj1pqe

### CODE REVIEW

File: app/assets/v2/css/grants/grant.css [hunk 1]
```diff
@@ -387,6 +387,14 @@
   color: white;
 }
 
+.grants-hero .title-round{
+  font-size: 24px; 
+}
+
+.grants-hero .amount-round{
```
Comment (thelostone-mc) [inline app/assets/v2/css/grants/grant.css:394]: .grants-hero .amount-round{ -> .grants-hero .amount-round {


same for above

File: app/retail/templates/shared/grant_thanks_modal.html [hunk 2]
```diff
@@ -49,9 +49,10 @@ <h5>Thanks for Contributing! You Earned a Kudos</h5>
                 {% trans "Tweet About this Grant" %} 🐦
               </a>
             {% else %}
-              <a class="button button--primary" role="button" href="https://twitter.com/intent/tweet?text=I just funded grants on @gitcoin => https://{{ request.META.HTTP_HOST }}grants"
```
Comment (thelostone-mc) [inline app/retail/templates/shared/grant_thanks_modal.html:52]: Shouldn't this still be in here ?

**Comment by developerfred** [comment]
done @thelostone-mc 👍

File: app/retail/templates/shared/grant_thanks_modal.html [hunk 3]
```diff
@@ -42,16 +42,12 @@ <h5>Thanks for Contributing! You Earned a Kudos</h5>
...
-              <a class="button button--primary" role="button" href="https://twitter.com/intent/tweet?text=I just funded this grant on @gitcoin => https://{{ request.META.HTTP_HOST }}/{% url 'grants:details' grant.id grant.slug %}"
-                target="_blank" rel="noopener noreferrer" OnClick="ga('send', 'event', 'Tweet Button', 'click', 'Grant Funder')">
-                {% trans "Tweet About this Grant" %} 🐦
-              </a>
+              </a> 
             {% else %}
-              <a class="button button--primary" role="button" href="https://twitter.com/intent/tweet?text=I just funded grants on @gitcoin => https://{{ request.META.HTTP_HOST }}/grants"
-                target="_blank" rel="noopener noreferrer" OnClick="ga('send', 'event', 'Tweet Button', 'click', 'Grant Funder')">
-                {% trans "Tweet About It" %} 🐦
+              <a class="button button--primary" role="button"
```
Comment (thelostone-mc) [inline app/retail/templates/shared/grant_thanks_modal.html:47]: Was  the tweet button intentionally removed ?

**Comment by danlipert** [comment]
Hey @developerfred - you need to fill out the testing section of the PR description when you submit a PR - let me know if you dont understand this as I see a lot of PRs from you with this section blank

File: app/retail/templates/shared/grant_thanks_modal.html (hunk 3 above)
Comment (danlipert) [reply app/retail/templates/shared/grant_thanks_modal.html:47]: @PixelantDesign we probably want both the tweet button as well as the fund a new grant button, correct?

**Comment by developerfred** [comment]
@danlipert Thank  you for informing me that it is a priority field.

File: app/retail/templates/shared/grant_thanks_modal.html (hunk 3 above)
Comment (octavioamu) [reply app/retail/templates/shared/grant_thanks_modal.html:47]: mmm yes tweet seems was something people like. Unless @PixelantDesign asked to remove this, lets get it back.

Comment (developerfred) [reply app/retail/templates/shared/grant_thanks_modal.html:52]: Done!

**Pull request merged**